src/main/java/ca/corefacility/bioinformatics/irida/ria/web/services/UIAdminStatisticsService.java

```diff
@@ -86,17 +86,21 @@ public StatisticsResponse getAdminAnalysesStatistics(Integer timePeriod) {
 
 		if (IntStream.of(StatisticTimePeriod.DAILY.getDaily())
```

Comment:
If you ended up moving the `GroupByFormat` stuff into `StatisticsResponse` you could also refactor to remove this whole `if` stack and get the method to just figure out which `StatisticsResponse` to use, and directly call into the service method with that `StatisticsResponse`.

Reply:
Updated in c5a00bcd1e78198ca591eaa499e2e494dd1cd5ca

Reply:
Thanks.  That paired down all those functions by a pile.